### PR TITLE
First implementation of deleting old logs

### PR DESCRIPTION
**Pull request by The1Penguin**

### CODE REVIEW

**Review by Multipacker — APPROVED** [review]
Looks good and I can actually kinda understand this one.

File: app/Main.hs [hunk 1]
```diff
@@ -53,9 +53,10 @@ import           View                                     ( render )
 
 opts :: [OptDescr (Config -> Config)]
 opts =
-  [ Option [] ["help"] (NoArg (set cHelp True))           "Show usage info"
-  , Option [] ["port"] (ReqArg (set cPort . read) "PORT") "Port to run on"
-  , Option [] ["path"] (ReqArg (set cLogPath) "PATH")     "Path to save log files to, default is 'logs'"
+  [ Option [] ["help"] (NoArg (set cHelp True))            "Show usage info"
```
Comment (Rembane) [inline app/Main.hs]: Making tables <3 

```suggestion
  [ Option [] ["help"] (NoArg  (set cHelp True))           "Show usage info"
```

**Review by Rembane — APPROVED** [review]
Good stuff, it's sufficiently cursed, I've added some indentation and doc changes in a later commit, and there's a suggestion for one more moving-space-around-to-make-it-prettier-change in comments. Do with them what thou wilst!

**Review by Rembane — APPROVED** [review]
LambdaCase and do-notation, it's the good things in life. Lets get this online and see what happens! :D

**Pull request merged**